src/broken_spec/lib/FacetsResponse_spec.js

```diff
@@ -1,6 +1,6 @@
-import FacetsResponse from '../../lib/FacetsResponse';
+import FacetsResponse from '../../lib/FacetsResponse.js';
 
-describe('FacetsResponse', function () {
+describe.skip('FacetsResponse', function () {
```

Comment:
I had added all the `skips` before I moved them to the `broken_spec` directory to allow me to make the `test` job in npm simpler and easier to see what needed to be fixed.  So after moving them, I just left them how I changed them.  If you want, I can go back and remove all the skips.  I just thought it would be easier to leave them at that point.